src/main/kotlin/org/rust/lang/core/RsPsiPattern.kt

```diff
@@ -25,64 +25,64 @@ object RsPsiPattern {
     private val STATEMENT_BOUNDARIES = TokenSet.create(SEMICOLON, LBRACE, RBRACE)
 
     /**
-     * Source of attributes: [https://doc.rust-lang.org/1.30.0/reference/attributes.html]
+     * Source of attributes: [https://doc.rust-lang.org/1.41.1/reference/attributes.html#built-in-attributes-index]
      */
-    private val STD_ATTRIBUTES: Set<String> = setOf(
-        "crate_name",
-        "crate_type",
-        "no_builtins",
-        "no_main",
-        "no_start",
-        "no_std",
-        "recursion_limit",
-        "windows_subsystem",
+    val STD_ATTRIBUTES: Set<String> = setOf(
```

Comment:
For built-in attribute that have been removed in the new version (E.g `linked_from`).  keep it, but it is better to provide detection only for toolchains with this attribute?

Reply:
I don't think that there is a big difference because we don't have any checks related to such attributes, so we can use any version of this list currently